### DISCUSSION TITLE
The download page appears to be broken

### DISCUSSION THREAD

**Discussion by mbrijun** [ImageMagick/ImageMagick#2972] [Help]
Hi, the Download page seems to be broken on www.imagemagick.org. It returns an empty page. I get the same result with Firefox and Edge under Windows 10. I am not sure if this is the right place to report this issue. Thank you.

**Comment by fmw42** [comment]
It works fine for me. Perhaps clear your browser cache and try again.

**Comment by mbrijun** [comment]
Hi @fmw42. I thought that too, but 2 different browsers are unlikely to have the same problem. This is how it looks for me (the same situation in Firefox, Edge and Edge Private Mode).

![image](https://user-images.githubusercontent.com/22507450/101991960-fb677b00-3ca7-11eb-903d-b57fcb102d92.png)

**Comment by fmw42** [comment]
ImageMagick web site has been moved and restructured. The download page is at https://imagemagick.org/script/download.php

Note: no leading www.

However, now I am getting the same results as you.  I do not know what is going on. So perhaps there is some moving around going on right now. Give it some time and see if things get worked out. Try in a couple of hours.

**Reply by mbrijun** [reply]
Confirmed, this is the exact page I am trying to access.

**Reply by fmw42** [reply]
Perhaps they are building new releases

**Reply by mbrijun** [reply]
I will check again tomorrow. There is no immediate need for me to see that page, but I thought I would let the team know just in case they were not aware. Just checked from an Android phone, same problem. Definitely not a stale cache issue.

**Reply by fmw42** [reply]
It is working again, now.

**Comment by urban-warrior** [comment]
The problem was a transient permission problem.  You should have no problem viewing the download page now.
